docs/modules/ROOT/pages/how-tos/k3s_maintenance.adoc

```diff
@@ -0,0 +1,36 @@
+= K3s Maintenance
+
+== Kubernetes Maintenance
+
+The Kubernetes version is bound to the K3s version, so by updating K3s you also update the Kubernetes version.
+The https://github.com/rancher/system-upgrade-controller[System Upgrade Controller] from Rancher is used to upgrade the K3s version.
+The System Upgrade Controller is installed using a https://github.com/projectsyn/component-system-upgrade-controller[Project Syn component].
+
+In order to manage the K3s version the following System Upgrade Controller plan is used (the plan is also installed using the Project Syn component):
+
+[source,yaml]
+----
+include::example$suc_plan_k3s.yaml[]
+----
+
+Check the https://github.com/k3s-io/k3s/releases[K3s Release] page and update the value `version` with the version you want to upgrade to. Example: `version: v1.20.4+k3s1`
+
+This will immediatly trigger an upgrade of K3s. The container image `docker.io/rancher/k3s-upgrade` will replace the existing K3s binary and restart K3s.
+
+== Node Maintenance
+
+We use Ubuntu as the underlaying operating system for K3s Kubernetes clusters. Node maintenance is also done using the System Upgrade Controller. 
+To upgrade the system packages the following scripts in a container image are used: https://github.com/projectsyn/system-upgrade-controller-package-upgrade
+Every week a new container image is build with the updates os packages. This container image is used in the System Upgrade Controller plan.
+
+The following System Upgrade Controller plan is used (the plan is also installed using the Project Syn component):
+
+[source,yaml]
+----
+include::example$suc_plan_os.yaml[]
+----
+
+This plan uses a `channel` instead of the `version` argument. The Project Syn component configures this to the https://github.com/projectsyn/floodgate[Floodgate URL]: `https://floodgate.syn.vshn.net/window/<day>/<hour>`
```

Comment:
```suggestion
This plan uses a `channel` instead of the `version` argument.
The Project Syn component configures the channel to use https://github.com/projectsyn/floodgate[Floodgate] to schedule a weekly node maintenance window: `https://floodgate.syn.vshn.net/window/<day>/<hour>`
```